contracts/RMRK/access/Ownable.sol

```diff
@@ -98,6 +114,7 @@ contract Ownable is Context {
      */
     function revokeContributor(address contributor) external onlyOwner {
```

Comment:
I've been thinking this for a while. What if we had a unified method with a flag, to `updateContributor`.  We could save some valuable bytes there :eyes:

Reply:
I think this makes sense 👌 Let me do it and we'll see the benefits 😄

Reply:
Done ✅